Formula/minetest.rb

```diff
@@ -59,4 +64,18 @@ def caveats
       "/Applications/minetest.app/Contents/MacOS/minetest --server".
     EOS
   end
+
+  test do
+    #
+    # --help and --version produce output directly.
+    # --speedtests and --videomodes need user data directory in order to work.
+    # --info and --trace need user data directory and run the game.
+    #
+    # --run-unittests and --speedtests do not work for Homebrew.
+    #
+    # Debug File: all test information should wind up in here.
+    #
+    (testpath/"Library/Application Support/minetest/debug.txt").write("")
+    system "#{prefix}/minetest.app/Contents/MacOS/minetest", "--videomodes"
```

Comment:
Considering this uses a `.app`, I think it is better for this formula to be moved to Homebrew Cask.

https://docs.brew.sh/Acceptable-Formulae#stuff-that-builds-an-app